src/types/zoneddatetime.jl

```diff
@@ -28,9 +28,10 @@ end
     ZonedDateTime(dt::DateTime, tz::TimeZone; from_utc=false) -> ZonedDateTime
 
 Construct a `ZonedDateTime` by applying a `TimeZone` to a `DateTime`. When the `from_utc`
-keyword is true the given `DateTime` is assumed to be in UTC instead of in local time and is
-converted to the specified `TimeZone`.  Note that when `from_utc` is true the given
-`DateTime` will always exists and is never ambiguous.
+keyword is true the given `DateTime` is assumed to be in UTC and is converted to the specified
+`TimeZone`.  When `from_utc` is false the given `DateTime` is assumed to already be in the
+specified `TimeZone`. Note that when `from_utc` is true the given `DateTime` will always exist
+and is never ambiguous.
```

Comment:
Better? 
```suggestion
keyword is `true` the given `DateTime` is declared to be in UTC but will be displayed in the specified
`TimeZone`.  When `from_utc` is `false` the given `DateTime` is declared to be in local time, or in the
specified `TimeZone`. Note that when `from_utc` is `true` the provided UTC `DateTime` will always exist
and is never ambiguous.
```

Reply:
I think "will be displayed" leaks the abstraction; to the user, they don't need to know that the underlying storage is in UTC. I can see why you would want to avoid "converted" though. I'm struggling to come up with good wording.

I think the phrase "local time" should be avoided to avoid confusion with `localzone()`/the OS time zone.

Reply:
> I think "will be displayed" leaks the abstraction; to the user, they don't need to know that the underlying storage is in UTC. I can see why you would want to avoid "converted" though. I'm struggling to come up with good wording.

I'm definitely up for suggestions. I think this part of the documentation has been hard to get right.

> I think the phrase "local time" should be avoided to avoid confusion with `localzone()`/the OS time zone.

The phrase "local time" is something the [IANA documentation](https://data.iana.org/time-zones/tz-how-to.html) uses and isn't quite perfect as really it just refers to only the time component. Maybe "local timestamp" or "local `DateTime`" is an improvement.

I suspect we'll have to iterate on this a little bit. Thank you for opening the PR :)